resources/assets/coffee/react/_components/comment.coffee

```diff
@@ -218,6 +222,25 @@ export class Comment extends React.PureComponent
           osu.trans('common.buttons.delete')
 
 
+  renderDeletedBy: =>
+    if @props.comment.isDeleted && @props.comment.canModerate
+      div className: 'comment__row-item comment__row-item--info',
+        el StringWithComponent,
+          pattern: osu.trans('comments.deleted_by')
+          mappings:
+            ':timeago':
+              el TimeWithTooltip,
+                key: 'timeago'
+                dateTime: @props.comment.deletedAt
+                relative: true
+            ':user':
+              if @props.comment.deletedById?
+                deleter = userStore.get(@props.comment.deletedById)
```

Comment:
can use a `? deletedUser`

Reply:
Also related to https://github.com/ppy/osu-web/pull/7744#discussion_r656816575

Maybe this can become something like this?

```
if @props.comment.deletedById?
    deleter = userStore.get(@props.comment.deletedById)
    if deleter? then el(UserLink, key: 'user', user: deleter) else deletedUser
```

Reply:
not sure how that's better than just `el UserLink, key: 'user', user: (userStore.get(...) ? deletedUser)`

Reply:
Oh, I thought `deletedUser` is just a string. Okay now this makes sense